addons/website/static/src/snippets/s_hr/000.scss

```diff
@@ -4,7 +4,8 @@
     hr {
         padding: 0;
         border: 0;
-        border-top: 1px solid currentColor;
+        border-top: 1px solid;
+        border-color: var(--o-border-color);
```

Comment:
Not sure if it was introduced by me, but: 
`border-top: 1px solid var(--o-border-color);`